en/dev_setup/dev_env_mac.md

```diff
@@ -65,12 +65,12 @@ brew install px4-sim-gazebo
 
 ## jMAVSim Simulation
 
-To use SITL simulation with jMAVSim you need to install a recent version of Java (e.g. Java 14).
-You can either download [Java 14 from Oracle](https://www.oracle.com/java/technologies/javase-jdk14-downloads.html) or use the AdoptOpenJDK tap:
+To use SITL simulation with jMAVSim you need to install the most recent version of Java (e.g. Java 15).
+You can either download [Java 15 from Oracle](https://www.oracle.com/java/technologies/javase-downloads.html) or use the AdoptOpenJDK tap:
```

Comment:
As @hamishwillee says, whatever comes with Ubuntu 20.04 (which is what we're trying to support) should be fine.